game.py

```diff
@@ -20,7 +20,29 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print_snowman_graphic(wrong_guesses_count)
+        
+        letter_guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if letter_guess in snowman_word:
+            for letter in snowman_word:
+                if letter == letter_guess:
+                    correct_letter_guess_statuses[letter] = True
```

Comment:
Do we actually need to preform this logic? Since `if letter_guess in snowman_word` determines if the guess was inside of our `snowman_word` then couldn't we also just do this instead? 

`correct_letter_guess_statuses[letter_guess] = True`